packages/ckeditor5-ui/src/toolbar/toolbarview.ts

```diff
@@ -410,7 +427,7 @@ export default class ToolbarView extends View implements DropdownPanelFocusable
 					 * @error toolbarview-item-unavailable
 					 * @param {String} name The name of the component.
 					 */
-					logWarning( 'toolbarview-item-unavailable', { name } );
+					logWarning( 'toolbarview-item-unavailable', { name: item } );
```

Comment:
Is this still a name only? Maybe we should pass the item there and update the param above?

Reply:
It's not only a name anymore. Do you want to make it `item`?

Reply:
> It's not only a name anymore. Do you want to make it `item`?

👍  and please remember to update the param in jsdoc above